src/Components/NavBarTest/Menus/NotificationsMenu.tsx

```diff
@@ -0,0 +1,180 @@
+      context_module: AnalyticsSchema.ContextModule.HeaderActivityDropdown,
+      destination_path: href,
+      subject,
+    })
+  }
+
+  return (
+    <>
+      {notifications.map(({ node }, index) => {
+        const { artists, href, image, summary } = node
+        return (
+          <MenuItem
+            href={href}
+            key={index}
+            onClick={() => {
+              handleClick(href, AnalyticsSchema.Subject.Notification)
+            }}
+          >
+            <Flex alignItems="center">
+              <Box width={40} height={40} bg="black5" mr={1}>
+                <Image
+                  src={image.resized.url}
+                  width={40}
+                  height={40}
+                  style={{
+                    objectFit: "cover",
+                  }}
+                />
+              </Box>
+              <Box>
+                <Sans size="2">{summary}</Sans>
+                <Sans size="2" weight="medium">
+                  {artists}
+                </Sans>
+              </Box>
+            </Flex>
+          </MenuItem>
+        )
+      })}
+
+      <Flex py={1} flexDirection="column" alignItems="center">
+        <>
+          {notifications.length === 0 && (
+            <Flex width="100%" flexDirection="column">
+              <Box pt={1} pb={3} width="100%" textAlign="center">
+                <Serif size="3">No new works</Serif>
+              </Box>
+            </Flex>
+          )}
+
+          <Box width="100%" px={2}>
+            <Separator />
+          </Box>
+
+          <Box pt={2}>
+            <Sans size="2">
+              <Link
+                href="/works-for-you"
+                onClick={() => {
+                  handleClick("/works-for-you", AnalyticsSchema.Subject.ViewAll)
+                }}
+              >
+                View all
+              </Link>
+            </Sans>
+          </Box>
+        </>
+      </Flex>
+    </>
+  )
+}
+
+/**
+ * The <Menu /> component renders a QueryRenderer inside of it, which fetches
+ * individual MenuItems for display. During fetch there is a loading spinner.
+ */
+export const NotificationsMenu: React.FC = () => {
+  return (
+    <Menu title="Activity">
+      <NotificationsQueryRenderer
+        render={renderWithLoadProgress(
+          NotificationMenuItems,
+          {},
+          {},
+          { size: "small" }
+        )}
+      />
+    </Menu>
+  )
+}
+
+/**
+ * This QueryRenderer is also shared with NotificationBadge. Once the request
+ * is performed the data is cached at the network layer so menu data appears
+ * immediately and doesn't require a second request.
+ */
+export const NotificationsQueryRenderer: React.FC<{
+  render: LoadProgressRenderer<any>
+}> = ({ render }) => {
+  const { relayEnvironment } = useContext(SystemContext)
+
+  return (
+    <QueryRenderer<NotificationsMenuQuery>
+      environment={relayEnvironment}
+      query={graphql`
+        query NotificationsMenuQuery {
```

Comment:
I think the re-declaration of the this query with the same name as the one in the _existing_ `NavBar` component is causing a legit relay failure in CI.